src/App.js

```diff
@@ -1,122 +1,99 @@
-                    className="search-input"
-                    type="text"
-                    onChange={(event) => {
-                        setPokemonName(event.target.value);
-                    }}
-                ></input>
-                <button onClick={searchPokemon} className="search-button">
-                    Search Pokemon
-                </button>
-            </div>
-            <div className="display-section">
-                {!isChosen ? (
-                    <h1>Please choose a pokemon</h1>
-                ) : (
-                    <div className="stats-window">
-                        <h1 className="stats-title">{pokemonInfo.name}</h1>
-                        <div className="text-and-image-wrapper">
-                            <div className="text-stats">
-                                <p>
-                                    <span className="bold">Species:</span>{" "}
-                                    {pokemonInfo.species}
-                                </p>
-                                <p>
-                                    <span className="bold">Type:</span>{" "}
-                                    {pokemonInfo.type}
-                                </p>
-                                <p>
-                                    <span className="bold">HP:</span>{" "}
-                                    {pokemonInfo.hp}
-                                </p>
-                                <p>
-                                    <span className="bold">Attack:</span>{" "}
-                                    {pokemonInfo.attack}
-                                </p>
-                                <p>
-                                    <span className="bold">Defence:</span>{" "}
-                                    {pokemonInfo.defense}
-                                </p>
-                            </div>
-                            <div className="image-window">
-                                <img
-                                    className="pokemon-image"
-                                    src={pokemonInfo.imageUrl}
-                                    alt="front view of chosen pokemon"
-                                ></img>
-                            </div>
-                        </div>
-                    </div>
-                )}
-            </div>
-        </div>
-    );
+  }, []);
+
+  const handleChosen = () => {
+    if (pokemonName) {
+      setIsChosen(true)
+    }
+  }
+
+  useEffect(() => {
+    if (location.pathname !== "/") {
+      setIsChosen(true)
+    } else {
+      setIsChosen(false);
+    }
+  }, [location.pathname]);
+
+  return (
+    <div className="App">
+      <div className="title-section">
+        <h1 className="title">Poke Stats</h1>
+        <input
+          className="search-input"
+          type="text"
+          onChange={(event) => {
+            setPokemonName(event.target.value);
+          }}
+        ></input>
```

Comment:
nit: if this doesn't have content inside it you can just make it self closing. (which for `input` is going to nearly always be true 'cause it doesn't accept children)

```suggestion
        <input
          className="search-input"
          type="text"
          onChange={(event) => {
            setPokemonName(event.target.value);
          }}
        />
```